src/common/darktable.c

```diff
@@ -160,13 +164,13 @@ gboolean dt_supported_image(const gchar *filename)
 static void strip_semicolons_from_keymap(const char *path)
 {
   char pathtmp[PATH_MAX] = { 0 };
-  FILE *fin = fopen(path, "r");
+  FILE *fin = fopen(path, "rb");
```

Comment:
As far as I understood this part fo the code is using the default keymap file which is the result of the `gtk_accel_map_save` call. That result is always binary (unix style line endings), the file name is `keyboardrc_default`. Subsequently the code uses `gtk_accel_map_load` and `gtk_accel_map_save` libary functions to process the file. I'm not sure whether those GTK library functions can handle Windows line endings.